src/pages/About/Staff.vue

```diff
@@ -124,6 +131,14 @@ export default {
         role: ["Henlo"],
         face: "https://bulma.io/images/placeholders/1280x960.png",
         icon: "https://bulma.io/images/placeholders/96x96.png"
+      },
+      {
+        id: 3033,
+        name: "Raymond Huang",
+        handle: "rayh",
+        role: ["Staff", "sudo rm -rf /"],
```

Comment:
Remove "staff" from "role" in your JSON, since we only want you showing up once

Reply:
Updated